observation/mapper.go

```diff
@@ -30,12 +32,13 @@ func (mapper *Mapper) Map(row string, instanceID string) (*Observation, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	header := inputcsv.NewHeader(headerRow)
 	dimensionOffset, err := header.DimensionOffset()
 	if err != nil {
+		log.ErrorC("Dimension offset error", err, nil)
 		return nil, err
 	}
+	log.Info("This far1", nil)
```

Comment:
Assume this was from trying to work out what was happening in your test? Can be removed.